DropboxLike.Domain/Configuration/AwsConfiguration.cs

```diff
@@ -2,8 +2,8 @@ namespace DropboxLike.Domain.Configuration;
 
 public class AwsConfiguration
 {
-  public string AwsAccessKey { get; set; }
-  public string AwsSecretAccessKey { get; set; }
-  public string BucketName { get; set; }
-  public string Region { get; set; }
+  public string? AwsAccessKey { get; set; }
```

Comment:
None of these should be nullable since the application cannot start without secrets. An exception should be thrown when the environment variables come back null/are not present.